reactor-aeron-core/src/main/java/reactor/aeron/MessagePublication.java

```diff
@@ -180,7 +179,7 @@ void close() {
       logger.warn("{} failed on aeron.Publication close(): {}", this, ex.toString());
       throw Exceptions.propagate(ex);
     } finally {
-      disposePublishTasks();
+      disposeProcessors();
       onDispose.onComplete();
```

Comment:
let's repeat `disposeProcessors();` after this to close all subscribers

Reply:
How many times _repeat_ ? 1 ?

Reply:
only once

Reply:
I propose to create and issue in reactor aeron and assign this issue on me. In a meantime let's move on with relaese of the new version.

Reply:
> only once

may be 3 ? to be 3x times sure ?

Reply:
:)

Reply:
Are you kidding? so how will you complete and release resources and subscriber? 
so PR has some obvious bugs - pls fix it

Reply:
>> may be 3 ? to be 3x times sure ?

after the first invocation of `disposeProcessors();` you are invoking  `onDispose.onComplete();` so after this, you can you simple check like `onDispose.isDispose` to verify you can pass incoming requests or not.